src/modules/hip/kernel/pre_emphasis_filter.hpp

```diff
@@ -0,0 +1,74 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+__device__ void pre_emphasis_filter_hip_compute(d_float8 *src1_f8, d_float8 *src2_f8, d_float8 *dst_f8, float4 *coeff_f4)
+{
+    dst_f8->f4[0] = src1_f8->f4[0]  - *coeff_f4 * src2_f8->f4[0];
+    dst_f8->f4[1] = src1_f8->f4[1]  - *coeff_f4 * src2_f8->f4[1];
+}
+
+__global__ void pre_emphasis_filter_tensor(float *srcPtr,
+                                           uint srcStride,
+                                           float *dstPtr,
+                                           uint dstStride,
+                                           Rpp32s *srcLengthTensor,
```

Comment:
In HIP kernels, we need to have the parameters of base type like int, float, uchar
instead of RPP typedefs for these. So please change Rpp32s to int

For RpptAudioBorder you can use a int may be?
reinterpret as int in the kernel launch and pass it here

@r-abishek 
Can you please share your thoughts on how to handle this enum which is of RpptAudioBorderType?

Reply:
changed RPP32s to int